instrumentation/opentelemetry_redix/lib/opentelemetry_redix.ex

```diff
@@ -63,6 +63,22 @@ defmodule OpentelemetryRedix do
       |> Map.merge(net_attributes(connection))
       |> Map.merge(redix_attributes(meta))
 
+    parent_context =
+      case OpentelemetryProcessPropagator.fetch_ctx(self()) do
```

Comment:
Isn't this just the same as checking `OpenTelemetry.Ctx.get_current()`?

Reply:
I see it returns undefined instead of an empty context.

Reply:
Yeah, this is the same behaviour as `opentelemetry_ecto`